website/LICENCE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need it here since we already have one on the root?

Reply:
oh nah we can remove it